src/main/java/com/bootcamp/demo/model/component/ScooterStatus.java

```diff
@@ -6,11 +6,7 @@
 
     private final String value;
 
-    ScooterStatus(String value) {
-        this.value = value;
-    }
+    ScooterStatus(String value) { this.value = value; }
```

Comment:
The implementation of this & `getValue` could remain as they were, on separate lines.